google/resource_cloudfunctions_function.go

```diff
@@ -180,6 +188,13 @@ func resourceCloudFunctionsFunction() *schema.Resource {
 				ValidateFunc: validation.StringInSlice(allowedIngressSettings, true),
 			},
 
+			"vpc_connector_egress_settings": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Default:      functionDefaultVpcConnectorEgressSettings,
```

Comment:
```suggestion
```

See above- this defaults to unset in some cases, and `PRIVATE_RANGES_ONLY` in others.